docs/sources/enterprise/request-security.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Request security"
+description = "Grafana Enterprise request security"
+keywords = ["grafana", "security", "enterprise"]
+weight = 110
++++
+
+# Request security
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+Request security introduces ways to limit requests from the Grafana server, and it targets requests that are generated by users.
+
+Example:
+- data source metric queries
+- alert notifications
+
+This can be used to limit access to internal systems that the server Grafana runs on can access but that users of Grafana should not be able to access. This feature does not affect traffic from the Grafana users browser.
+
+Note: Although request security works with backend plugins, it is possible to create a backend plugin that bypasses this security.
+
+## IP and hostname blocking
+
+It is possible to limit requests based on a hostname, an IP address, or both.
+
+### Deny list
+
+Any request to a hostname or IP address on the deny-list is denied.
```

Comment:
```suggestion
Grafana blocks any request to a hostname or IP address on the deny list.
```

Reply:
I used "deny"  because it matches "deny / allow list verbiage. Agree with passive --> active voice. Also, block does read better imo.

Reply:
I feel like "The deny list denies stuff" doesn't add much information, so I try to vary verbs a bit. "The deny list blocks stuff that is on it" actually explains something.